src/common/transformations/src/transformations/low_precision/mark_dequantization_subgraph.cpp

```diff
@@ -309,3 +309,66 @@ ov::pass::KeepConstPrecision::KeepConstPrecision(const element::TypeVector& prec
     auto m = std::make_shared<Matcher>(multiply_pattern, "KeepConstPrecision");
     this->register_matcher(m, callback);
 }
+
+ov::pass::KeepDequantizationPrecision::KeepDequantizationPrecision(const element::TypeVector& precisions) {
+    MATCHER_SCOPE(KeepDequantizationPrecision);
+
+    auto input_pattern = pattern::any_input(pattern::type_matches_any(precisions));
+    auto convert_pattern = pattern::wrap_type<v0::Convert>({input_pattern}, pattern::consumers_count(1));
+
+    // zero points:
+    auto zp_pattern = pattern::any_input();
+    auto zp_convert_pattern = pattern::optional<v0::Convert>(zp_pattern);
+    auto zp_reshape_pattern = pattern::optional<v1::Reshape, v0::Unsqueeze>({zp_convert_pattern, pattern::any_input()});
+    auto subtract_pattern = pattern::optional<v1::Subtract>({convert_pattern, zp_reshape_pattern});
+
+    // scale:
+    auto scale_pattern = pattern::any_input();
+    auto scale_convert_pattern = pattern::optional<v0::Convert>(scale_pattern);
+    auto scale_reshape_pattern =
```

Comment:
very minor (optional): Single line alignment would be more elegant 😄

Reply:
Applied, thanks